staff/playwright.config.ts

```diff
@@ -0,0 +1,129 @@
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 5000,
+    toHaveScreenshot: {
+      /* Set a default ratio accept rate.
+       * GitHub Actions runs on Linux, and we're faking those screenshots
+       * by copying the Darwin (macos) ones
+       */
+
+      /* Another option is to have a command to update the screenshots from Github with
+       * a specific command:
+       * https://github.com/basarat/demo-playwright-vrt/blob/main/.github/workflows/update-snapshots.yml
+       */
+      maxDiffPixelRatio: 0.01,
+    },
+  },
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: "html",
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
+    actionTimeout: 0,
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://localhost:3000',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: "on-first-retry",
+
+    screenshot: "only-on-failure",
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: "chromium",
+      use: {
+        ...devices["Desktop Chrome"],
+      },
+    },
+
+    {
+      name: "firefox",
+      use: {
+        ...devices["Desktop Firefox"],
+      },
+    },
+
+    {
+      name: "webkit",
+      use: {
+        ...devices["Desktop Safari"],
+      },
+    },
+
+    /* Test against mobile viewports. */
+    // {
+    //   name: 'Mobile Chrome',
+    //   use: {
+    //     ...devices['Pixel 5'],
+    //   },
+    // },
+    // {
+    //   name: 'Mobile Safari',
+    //   use: {
+    //     ...devices['iPhone 12'],
+    //   },
+    // },
+
+    /* Test against branded browsers. */
+    // {
+    //   name: 'Microsoft Edge',
+    //   use: {
+    //     channel: 'msedge',
+    //   },
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: {
+    //     channel: 'chrome',
+    //   },
+    // },
+  ],
+
+  /* Folder for test artifacts such as screenshots, videos, traces, etc. */
+  // outputDir: 'test-results/',
+
+  /* Run your local dev server before starting the tests */
+  /* This command is what gets run by playwright automatically
+   * (and implicitly when you run `npm run e2e`)
+   */
+  webServer: {
+    command:
+      "npm run prisma:update && \
+       docker compose up --build && \
+       docker compose exec remix_dev npx prisma migrate reset --force",
+    port: 3033,
```

Comment:
Will your lowdefy app need the database for testing? 
and
I'm pretty sure this isn't the command to start lowdefy?

Reply:
> Will your lowdefy app need the database for testing? 
Yes. I refactored the playwright config to use `docker-compose.yml` so that `docker compose up --build` command will start the postgres.

> and I'm pretty sure this isn't the command to start lowdefy?
The `docker compose up --build` command will start lowdefy